bookkeeper-server/src/test/proto/TestDataFormats.proto

```diff
@@ -30,5 +30,5 @@ enum AuthMessageType {
  *
  */
 extend AuthMessage {
-    required AuthMessageType messageType = 1000;
+    optional AuthMessageType messageType = 1000;
```

Comment:
@eolivelli protobuf 2.6 no longer allows ading required fields to extended messages. Protoc errors out.
Please see this conversation for details: https://groups.google.com/forum/#!topic/protobuf/EyR1EdDL5cs 
"Extensions, by its definition, is something optionally attached to a message and it doesn't make sense for it to be required, and technically there is no way to enforce this requirement because a binary can be compiled without the extension definition in which case there is no way to know whether the extension is required or optional.

And the change in 2.6.0 is actually a bug fix because in previous versions the required label is simply ignored and the extension is treated as optional any way."